docs/src/modules/utils/find.js

```diff
@@ -23,7 +23,7 @@ function findPagesMarkdown(
     }
     let pathname = itemPath.replace(new RegExp(`\\${path.sep}`, 'g'), '/').replace(/^.*\/pages/, '').replace('.md', '');
 
-    if (pathname.indexOf('/demos') === 0) {
+    if (pathname.indexOf('/demos') === 0 || pathname.indexOf('/lab') === 0 || pathname.indexOf('/utils') === 0 ) {
```

Comment:
I will try to find a better approach than having to explicit the folders that follow a different structure in order to fix #10337.